Dockerfile

```diff
@@ -21,17 +21,16 @@ ARG LOG_CONFIG=assets/production_log_config.json
 ENV LOG_CONFIG=${LOG_CONFIG}
 
 # Updgrade system packages and install curl
-RUN apt-get update && apt-get upgrade -y && apt-get install curl -y
+RUN apk update && apk upgrade && apk add build-base python3-dev --no-cache linux-headers curl
```

Comment:
@ericbuckley I found that I needed linux-headers also installed to get it fully working locally